lua/lualine/components/signify.lua

```diff
@@ -1,33 +1,73 @@
--- Copyright (c) 2020-2021 hoob3rt
+-- Copyright (c) 2020-2021 shadmansaleh
 -- MIT license, see LICENSE for more details.
 
-local function signify()
-   if vim.fn.exists('*sy#repo#get_stats') == 0 then return '' end
-   local added, modified, removed = unpack(vim.fn['sy#repo#get_stats']())
-   if added == -1 then return '' end
-   local symbols = {
-     '+',
-     '-',
-     '~',
-   }
-   local result = {}
-   local data = {
-    added,
-    removed,
-    modified,
-   }
-   for range=1,3 do
-     if data[range] ~= nil and data[range] > 0
-       then table.insert(result,symbols[range]..''..data[range]..' ')
-     end
-   end
-
-   if result[1] ~= nil then
-       return table.concat(result, '')
-   else
-       return ''
-   end
-end
+local utils = require'lualine.utils.utils'
+
+local function signify(options)
+  local colored = true
+  if options.colored ~= nil then colored = options.colored end
```

Comment:
I would change this to look cleaner. 
```lua
if options.colored == nil then options.colored = true end
```
and work with options variables. Similar to all other option variables. 
This has two benefits
* while looking at the code you instantly know that `if options.colored` refers to an option. `if colored` can be a local variable
* no additional memory usage

Reply:
good idea . I'll change that